torchaudio/io/_stream_reader.py

```diff
@@ -182,24 +174,95 @@ def _get_vfilter_desc(frame_rate: Optional[float], width: Optional[int], height:
         scales.append(f"height={height}")
     if scales:
         descs.append(f"scale={':'.join(scales)}")
-    if format is not None:
-        fmt = {
-            "RGB": "rgb24",
-            "BGR": "bgr24",
-            "YUV": "yuv420p",
-            "GRAY": "gray",
-        }[format]
+    if fmt is not None:
         descs.append(f"format=pix_fmts={fmt}")
     return ",".join(descs) if descs else None
 
 
+def _format_doc(**kwargs):
+    def decorator(obj):
+        obj.__doc__ = obj.__doc__.format(**kwargs)
+        return obj
+
+    return decorator
+
+
+_frames_per_chunk = """Number of frames returned as one chunk.
+                If the source stream is exhausted before enough frames are buffered,
+                then the chunk is returned as-is."""
+
+_buffer_chunk_size = """Internal buffer size.
```

Comment:
could you add default args to the optional parameters as well?

Reply:
done!